### ISSUE TITLE
--deploy_parameters option is present when no deploy parameters are set 

### ISSUE THREAD

**Issue by lukebarge** [google-github-actions/create-cloud-deploy-release#54]
### TL;DR

`--deploy_parameters` option is present when no deploy parameters are set. The `gcloud` fails because there is no argument provided to it 

### Expected behavior

When no deploy parameters are provided, there should be no `--deploy_parameters` in the output command

### Observed behavior

When no deploy parameters are provided,  `--deploy_parameters` is present in the output command but without an argument passed to it. This causes the `gcloud` to fail.

The cause is https://github.com/google-github-actions/actions-utils/blob/9c1562ded0e506d4d957f6e3a56fb1dbfa6cb59b/src/kv.ts#L89 which returns an empty object if there are no arguments provided which evaluates truthy here https://github.com/google-github-actions/create-cloud-deploy-release/blob/fd4cd1156f42bb00467465370c9d7ba5790b1ab1/src/main.ts#L93

### Action YAML

```yaml
name: Build app and create a release in Cloud Deploy

on:
  push:
    branches:
      - $default_branch

env:
  PROJECT_ID: nifty-might-418012 
  GAR_LOCATION: europe-west2 
  REGION: europe-west2 
  APP: app

jobs:
  deploy:
    permissions:
      contents: 'read'
      id-token: 'write'

    runs-on: ubuntu-latest
    steps:
      - name: 'Checkout'
        uses: 'actions/checkout@v3'

      - name: 'Google auth'
        id: 'auth'
        uses: 'google-github-actions/auth@v2'
        with:
          credentials_json: '${{ secrets.GCP_SA_KEY }}' # Replace with the name of your GitHub Actions secret


      - name: 'Set up Cloud SDK'
        uses: 'google-github-actions/setup-gcloud@v1'
        with:
          project_id: '${{ env.PROJECT_ID }}'

      - name: 'Docker auth'
        run: |-
          gcloud auth configure-docker ${{ env.GAR_LOCATION }}-docker.pkg.dev

      - name: 'Build and push container'
        run: |-
          docker build -t "${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.PROJECT_ID }}/${{ env.APP }}/${{ env.APP }}:${{ github.sha }}" .
          docker push "${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.PROJECT_ID }}/${{ env.APP }}/${{ env.APP }}:${{ github.sha }}"

      - name: 'Render templatised config manifests'
        run: |-
          export PROJECT_ID="${{ env.PROJECT_ID }}"
          export REGION="${{ env.REGION }}"
          for template in $(ls config/*.template.yaml); do envsubst < ${template} > ${template%%.*}.yaml ; done

      - name: 'Create Cloud Deploy delivery pipeline'
        run: |-
          gcloud deploy apply --file config/clouddeploy.yaml --region ${{ env.GAR_LOCATION }}

      - name: 'Create release name'
        run: |-
          echo "RELEASE_NAME=${{ env.APP }}-${GITHUB_SHA::7}-${GITHUB_RUN_NUMBER}" >> ${GITHUB_ENV}

      - name: 'Create Cloud Deploy release'
        id: 'release'
        uses: 'google-github-actions/create-cloud-deploy-release@v0'
        with:
          delivery_pipeline: '${{ env.APP }}'
          name: '${{ env.RELEASE_NAME }}'
          region: '${{ env.REGION }}'
          description: '${{ env.GITHUB_COMMIT_MSG }}'
          skaffold_file: 'config/skaffold.yaml'
          images: 'app=${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.PROJECT_ID }}/${{ env.APP }}/${{ env.APP }}:${{ github.sha }}'

      - name: 'Report Cloud Deploy release'
        run: |-
          echo "Created release ${{ steps.release.outputs.name }} "
          echo "Release link ${{ steps.release.outputs.link }} "
```

### Log output

```text
Error: create-cloud-deploy-release failed with: failed to execute gcloud command `gcloud deploy releases create medi-quiz-92cb7cd-18 --delivery-pipeline medi-quiz --region europe-west2 --images app=europe-west2-docker.pkg.dev/nifty-might-418012/medi-quiz/medi-quiz:92cb7cd0342c1a956538d1b799eb8bc1009be6a2 --skaffold-file config/skaffold.yaml --deploy-parameters  --annotations commit=https://github.com/lukebarge/medi-quiz/commit/92cb7cd0342c1a956538d1b799eb8bc1009be6a2,git-sha=92cb7cd0342c1a956538d1b799eb8bc1009be6a2 --labels managed-by=github-actions --format json`: Creating temporary tarball archive of 45 file(s) totalling 539.6 KiB before compression.
```
